packages/pyroscope-flamegraph/src/Toolbar.spec.tsx

```diff
@@ -266,18 +266,23 @@ describe('ProfileHeader', () => {
     });
 
     it('updates to HEAD first', () => {
-      userEvent.selectOptions(
-        screen.getByRole('combobox', { name: /fit-mode/ }),
-        screen.getByRole('option', { name: /Head/ })
-      );
+      const toolbar = screen
+        .getByRole('toolbar')
+        .querySelector('.dropdownMenuButton') as HTMLElement;
```

Comment:
If we had to update these it means semantics have changed, which breaks accessibility https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role

Reply:
update the tests here https://github.com/pyroscope-io/pyroscope/pull/1435/commits/1469882669170e5443211a7e7db4496a9a23f60d to reflect the roles better